### PR TITLE
Gtw/refactor_docs

### PR DESCRIPTION
**Pull request by gregertw**
* Created new examples/ module dir
* Moved apm_simple into examples/
* Added readmes to modules
* Refactor all the docs

### CODE REVIEW

**Review by ronpal — APPROVED** [review]
Approved, with comments

**Review by doctrino — COMMENTED** [review]
Skimmed through and it looks good. I think, as discussed, we need a discussion on how to handle the configs.

**Review by doctrino — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,8 +74,6 @@ test framework for scenario based testing can be found in the Cognite private bi
 
 > TODO Define how to make sure that modules get tested in big-smoke.
 
-The `deploy.py` script will automatically clean configurations before trying to load, so you can
+The `deploy.py` script will clean configurations before trying to load if you specify `--drop`, so you can
```
[the remark this answers is not in the document]
Comment (gregertw) [reply CONTRIBUTING.md:77]: ? it does for me?

File: docs/advanced.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  you will have to adjust the template before applying it to your project.
+- Once a project is provisioned from these templates, you can continue to manage the governed parts of
+  your projects using the templates. This repo is a Github template, so you can start a new repository
+  based on this template and use it to manage your project. Your own modules should NOT have the `cdf_`
+  prefix as it is reserved for official Cognite product modules.
+- Currently, the scripts/ directory contains Python code based on the Cognite Python SDK that is
+  used to manage the project and the configurations are pushed to the various services in CDF as part
+  of the deploy step. Over time, it is expected that this deployment of individual configurations will
+  be replaced by CDF APIs that will allow lifecycle management of the configurations, as well as
+  offer validation and integrity checks.
```
[the remark this answers is not in the document]
Comment (gregertw) [reply docs/advanced.md:24]: No, you're right. Deleted

**Pull request merged**